run.py

```diff
@@ -685,6 +696,17 @@ def fetch_test_details(var) -> dict:
         "log_dir": run_dir,
         "run_id": run_id,
     }
+    download_path = run_dir if not log_directory else log_directory
+
+    # If Performance and CPU usage monitoring is enabled, perform pre-reqs
+    if enable_perf_mon:
```

Comment:
This won't work in case we have multisite or mirror cluster.

Reply:
Noted. The logic has been added under the exisitng for loop that iterates over the clusters